commons/src/config/locale.rs

```diff
@@ -60,11 +59,38 @@ pub mod serde {
         where
             D: Deserializer<'de>,
         {
-            let s = String::deserialize(deserializer)?;
+            deserializer.deserialize_seq(LocaleSeqVisitor {})
+        }
+
+        struct LocaleSeqVisitor {}
 
-            s.split(',')
-                .map(|lang| lang.parse().map_err(serde::de::Error::custom))
-                .collect()
+        impl<'de> Visitor<'de> for LocaleSeqVisitor {
+            type Value = HashSet<Locale>;
+            fn expecting(&self, formatter: &mut std::fmt::Formatter) -> std::fmt::Result {
+                formatter.write_str("Sequence of locales.")
+            }
+            fn visit_seq<M>(self, mut access: M) -> Result<Self::Value, M::Error>
+                where
+                    M: SeqAccess<'de>,
+            {
+                let mut set = HashSet::with_capacity(access.size_hint().unwrap_or(0));
+                while let Some(value) = access.next_element::<String>()? {
+                    match value.parse::<Locale>() {
+                        Ok(locale) => {
+                            set.insert(locale);
+                        }
+                        Err(e) => {
+                            log!(
+                                ConfigWarning::InvalidOutputLang,
+                                format!("Could not parse the output language to locale with error: '{}'", e)
+                            );
+                        }
+                    }
+                    let locale = value.parse::<Locale>().map_err(serde::de::Error::custom)?;
+                    set.insert(locale);
```

Comment:
```suggestion
                    match value.parse::<Locale>() {
                        Ok(locale) => {
                            set.insert(locale);
                        }
                        Err(e) => {
                            log!(
                                ConfigWarning::InvalidOutputLang,
                                format!("Could not parse the output language to locale with error: '{}'", e)
                            );
                        }
                    }
```

Otherwise, the locale would be inserted twice